src/server/db/seed/index.ts

```diff
@@ -1,57 +1,64 @@
-const { PrismaClient } = require("@prisma/client");
-const { users, habits, checkIns, contacts } = require('./seedData.json');
+import { PrismaClient } from "@prisma/client";
+import { users, habits, routines} from './seedData.json';
 const prisma = new PrismaClient();
 
+interface HabitSeedData {
+    name: string
+    datesCompleted: string[]
+    checkInDay: string
+    ownerId: number
+}
+
+const habitsWithDateObjects = habits.map((habit: HabitSeedData) => {
+    return ({
+        ...habit,
+        datesCompleted: habit.datesCompleted.map((date: string) => new Date(date)) 
+    })
+})
+
+console.log("habitsWithDateObjects: ", habitsWithDateObjects);
```

Comment:
Do you think we still need this console log for debugging? If not then I suggest deleting

Reply:
It was for verifying that the map method generated the object I wanted. I have now deleted the log.